environment/src/main/java/com/sequenceiq/environment/proxy/v1/converter/ProxyRequestToProxyConfigConverter.java

```diff
@@ -13,12 +14,16 @@
     public ProxyConfig convert(ProxyRequest source) {
         ProxyConfig proxyConfig = new ProxyConfig();
         proxyConfig.setName(source.getName());
-        proxyConfig.setPassword(source.getPassword());
         proxyConfig.setDescription(source.getDescription());
         proxyConfig.setProtocol(source.getProtocol());
         proxyConfig.setServerHost(source.getHost());
         proxyConfig.setServerPort(source.getPort());
-        proxyConfig.setUserName(source.getUserName());
+        if (StringUtils.isNoneBlank(source.getUserName())) {
+            proxyConfig.setUserName(source.getUserName());
+        }
+        if (StringUtils.isNoneBlank(source.getPassword())) {
+            proxyConfig.setPassword(source.getPassword());
+        }
```

Comment:
is there any reason to set if the other is blank? if not then it should look like:
`if (StringUtils.isNoneBlank(source.getUserName(), source.getPassword())) {
            proxyConfig.setUserName(source.getUserName());
			proxyConfig.setPassword(source.getPassword());
        }`
if yes, then you could use StringUtils.isNotBlank

Reply:
again....